docs/system-architecture.md

```diff
@@ -0,0 +1,48 @@
+# Licence
+
+<! --- SPDX-License-Identifier: CC-BY-4.0  -- >
+
+## System Architecture
+
+The system consists of 3 main components:
+
+- **Similar services component**, responsible for calculating the similarity between the onboarding service and the rest of the services in the catalogue.
```

Comment:
Similar services **Retriever** component, responsible for retrieving similar services to the onboarding service